### ISSUE TITLE
Is ASP.NET Core Identity a requirement?

### ISSUE THREAD

**Issue by Brice-xCIT** [skoruba/IdentityServer4.Admin#227]
Related to #79, I found while refactoring that ASP.NET Core Identity is pretty coupled with the admin services and controllers. I'm wondering, is Identity a hard requirement for the Admin UI, or could it be imaginable, from a design point of view, to have an Admin UI app without Identity. If so, what would it look like? Would there be fewer features, unavailable pages, etc?

**Comment by skoruba** [comment]
I will create a flag in `dotnet new` template for excluding Identity, but currently it is a requirement. 
- During instalation will be exluding all projects related to Identity, views and controllers. It will be necessary to create new instance of STS without Identity as well - with in-memory users?

**Comment by carlospence** [comment]
Any update on this feature

**Comment by skoruba** [comment]
Do you need remove asp.net core identity from project?

**Comment by carlospence** [comment]
Yes, I need to remove Asp.net core Identity from the project. I want to use my own User store bt still able to manage the clients
